.changeset/green-cycles-shake.md

```diff
@@ -0,0 +1,5 @@
+---
+'@sveltejs/kit': patch
+---
+
+fix: prevent esbuild adding phantom exports to service worker
```

Comment:
we should still remove the previous changeset

Reply:
i already did